spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Nit: can you remove this empty line?

Reply:
This should now be fixed.

Reply:
This blank line is still failing the build I think.

Reply:
@alexjfisher it should be fixed now.... a month later